packages/utils/test/unit/waitFor.test.ts

```diff
@@ -35,3 +36,29 @@ describe("waitFor", () => {
     await expect(waitFor(() => true, {interval, timeout, signal: controller.signal})).to.be.rejectedWith(ErrorAborted);
   });
 });
+
+describe("waitForElapsedTime", () => {
```

Comment:
Test case name does not match function name, looks like this was missed during renaming